fairlearn/widget/fairlearnDashboard.py

```diff
@@ -178,12 +187,20 @@ def _on_request(self, change):
     def _show(self):
         display(self._widget_instance)
 
+    def _sanitizeDataShape(self, dataset):
```

Comment:
nit: python doesn't do camel case, but 
_sanitize_data_shape

same in filenames btw